src/shared/parse-helpers.c

```diff
@@ -196,3 +198,47 @@ int parse_socket_bind_item(
         *port_min = mn;
         return 0;
 }
+
+int parse_open_file(const char *v, OpenFile* of) {
+        _cleanup_free_ char *path = NULL, *fdname = NULL, *flags = NULL;
+        int r;
+
+        assert(v);
+
+        r = extract_many_words(&v, ":", EXTRACT_DONT_COALESCE_SEPARATORS, &path, &fdname, &flags, NULL);
```

Comment:
This function will return the amount of arguments passed. If too few arguments are passed, the ones without a value are set to NULL. So we should verify that we received at least 1 argument for the path and error if we didn't get at least one word.